acceptancetests/assess_cross_model_relations.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+"""Functional tests for Cross Model Relation (CMR) functionality.
+
+This test exercises the CMR Juju functionality which allows applications to
+communicate between different models (including across controllers/clouds).
+
+The outline of this feature can be found here[1].
+
+This test will exercise the following aspects:
+  - Ensure a user is able to create an offer of a relation of an application
```

Comment:
An application on its own doesn't have a "relation". An application does define endpoints though, and an offer advertises one or more of those endpoints.